tests/test-class-amp-dom-utils.php

```diff
@@ -145,4 +145,58 @@ public function test_html5_empty_elements() {
 		$this->assertEquals( 'div', $video->childNodes->item( 4 )->nodeName );
 		$this->assertEquals( 'span', $video->childNodes->item( 5 )->nodeName );
 	}
+
+	/**
+	 * Get Table Row Iterations
+	 *
+	 * @return array An array of arrays holding an integer representation of iterations.
+	 */
+	public function get_table_row_iterations() {
+		return array(
+			array( 1 ),
+			array( 10 ),
+			array( 100 ),
+			array( 1000 ),
+			array( 10000 ),
```

Comment:
Per https://github.com/Automattic/amp-wp/issues/993#issuecomment-372139130 I think this 10,000 value should just be removed. I don't think it is reasonable to expect a `srcdoc` to be this large, let alone an AMP document as a whole.